subsys/nrf_security/src/drivers/cracen/cracenpsa/src/kmu.c

```diff
@@ -836,6 +836,13 @@ psa_status_t cracen_kmu_provision(const psa_key_attributes_t *key_attr, int slot
 	case CRACEN_KMU_KEY_USAGE_SCHEME_ENCRYPTED:
 #endif /* PSA_NEED_CRACEN_KMU_ENCRYPTED_KEYS */
 	case CRACEN_KMU_KEY_USAGE_SCHEME_RAW:
+		/* Since AES-keys can be stored using protected there is no reason to allow them to
```

Comment:
I agree with regard to the RAW scheme, with this we will not allow them to be stored encrypted as well. I don't have a strong opinion on this but I wonder if we still need to support the encrypted ones.

Reply:
Discussed this offline. Encrypted AES-keys so can be exported. Removed the block for encrypted keys.